src/Filament/Pages/EditSettings.php

```diff
@@ -64,5 +66,13 @@ public function save(): void
         });
 
         Setting::upsert($records, ['key'], ['value']);
+
+        // Clear the config cache
+        Artisan::call('config:clear');
```

Comment:
I believe that you can all `Cache::clear();`

Reply:
It turns out, that isn't what the artisan command does, and I wanted to exactly replicate that.